src/pubtools/sign/utils.py

```diff
@@ -20,3 +24,23 @@ def isodate_now():
     :return: str
     """
     return datetime.datetime.utcnow().isoformat() + "Z"
+
+
+def run_command(cmd, env=None):
```

Comment:
Does this function do anything?

Reply:
if you mean why it's extra function if it only does return call of one function with no preparation or anything like that, Then no, but I still think it's better to have it like this as you don't need to specify options you don't want to change